workers/dockerfiles/Dockerfile.affymetrix

```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:16.04
+
+# Much of this dockerfile is based off of the rpy2 docker image found:
+# https://hub.docker.com/r/rpy2/rpy2/~/dockerfile/
+
+RUN \
+  apt-get update -qq && \
+  apt-get install -y lsb-release && \
+  echo "deb http://cran.cnr.berkeley.edu/bin/linux/ubuntu $(lsb_release -sc)/" \
```

Comment:
I have been wondering why we used the mirror site at UCB. Wouldn't a mirror site on east coast (such as CMU or Duke) faster?
https://cran.r-project.org/mirrors.html